src/OrbitDB.js

```diff
@@ -75,6 +82,10 @@ class OrbitDB {
       options.storage = Storage(null, storageOptions)
     }
 
+    if (options.identity && options.identity.provider._keystore) {
```

Comment:
We should really expose `_keystore` as `keystore` in the identity provider. The member variables prefixed with `_` are not meant to be called directly (as in they are a notion of "private variables", as far as that goes in JS).

Reply:
Actually this is already implemented as a getter function in `orbit-db-identity-provider` so I'll just use that, np.